esrally/driver/runner.py

```diff
@@ -505,16 +505,14 @@ async def __call__(self, es, params):
         if "pipeline" in params:
             bulk_params["pipeline"] = params["pipeline"]
         if "refresh" in params:
-            refresh_params = {"sync": "wait_for", "async": "true", "default": "false"}
-            bulk_params["refresh"] = refresh_params.get(params["refresh"])
-
-            if bulk_params["refresh"] is None:
-                self.logger.warning(
-                    "Using default bulk refresh parameter value '%s' in place of unrecognized specfified parameter value '%s'. "
-                    "Use one of %s.",
-                    refresh_params["default"],
+            valid_refresh_values = ("wait_for", "true", "false")
+            if params["refresh"] in valid_refresh_values:
+                bulk_params["refresh"] = params["refresh"]
+            else:
+                self.logger.info(
```

Comment:
Sorry for my last comment, I see now what you're checking for. I think both `INFO` and `WARNING` aren't enough this case, and we should probably just error out because of an invalid value - otherwise we risk returning misleading results and the corresponding log message is buried deep in the file somewhere. 

Maybe we can error out similar to how it's done here?:
https://github.com/elastic/rally/blob/b1a822c19734cb350961eb15d34649ac24cda9e2/esrally/driver/runner.py#L2609-L2612

Reply:
We could probably also just lean on the client itself to return an error, if you pass something in that's not valid does it return anything meaningful?